libraries/core_libs/consensus/src/transaction/transaction_queue.cpp

```diff
@@ -77,13 +83,9 @@ bool TransactionQueue::erase(const trx_hash_t &hash) {
   queue_transactions_.erase(it);
 
   const auto &account_it = account_nonce_transactions_.find(it->second->getSender());
-  if (account_it == account_nonce_transactions_.end()) {
-    return false;
-  }
+  assert(account_it != account_nonce_transactions_.end());
   const auto &nonce_it = account_it->second.find(it->second->getNonce());
-  if (nonce_it == account_it->second.end()) {
-    return false;
-  }
+  assert(nonce_it != account_it->second.end());
 
   if (hash == nonce_it->second->getHash()) {
```

Comment:
btw should this be assert too ? Is it possible that these 2 values are different ?

Reply:
added the assert